arches/app/models/models.py

```diff
@@ -2383,6 +2384,37 @@ def __str__(self):
     class Meta:
         managed = True
         db_table = "spatial_views"
+        triggers = [
+            pgtrigger.Composer(
+                name="arches_update_spatial_views",
+                when=pgtrigger.After,
+                operation=pgtrigger.Update | pgtrigger.Delete | pgtrigger.Insert,
+                declare=[
+                    ("sv_perform", "text"),
+                    ("valid_geom_nodeid", "boolean"),
+                    ("has_att_nodes", "integer"),
+                    ("valid_att_nodeids", "boolean"),
+                    ("valid_language_count", "integer"),
+                ],
+                func=pgtrigger.Func(
+                    format_file_into_sql(
+                        "arches_update_spatial_views.sql",
+                        "sql/triggers",
+                    )
+                ),
+            )
+        ]
+
+    def clean_fields(self, exclude=None):
+        if exclude is not None:
```

Comment:
Need to call `super().clean_fields(exclude=exclude)` so we don't evade all the other checks. This has the advantage of ensuring that self.language and self.geometrynode don't crash below if absent (well, unless they were excluded, so we should probably still guard for that and raise ValidationError instead of letting another exception escape)